src/js/components/molecules/filter-category/filter-category.ts

```diff
@@ -0,0 +1,46 @@
+/** @format */
+
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { FilterCategoryProps } from './filter-category.models';
+import WithRender from './filter-category.vue.html';
+
+@WithRender
+@Component({
+    name: 'FilterCategory',
+})
+export class FilterCategory extends Vue implements FilterCategoryProps {
+    @Prop({ type: Array, required: true }) labels!: FilterCategoryProps['labels'];
+    @Prop({ type: String, required: true }) showAllLabel!: FilterCategoryProps['showAllLabel'];
+    @Prop({ type: String, required: true }) title!: FilterCategoryProps['title'];
+
+    get listItems(): object {
```

Comment:
Please do not create your own mappings. It creates a lot of confusion. Either loop in the vue.html and add the correct properties there or add it to the models. example: ``` v-for="labels"``` <InputCheckbox :label="" name="" />